pkg/store/filenames/filenames.go

```diff
@@ -52,6 +53,8 @@ const (
 	HostAgentStderrLog = "ha.stderr.log"
 	VzIdentifier       = "vz-identifier"
 	VzEfi              = "vz-efi"
+	WslRootFs          = "wsl-rootfs"
+	WslRootFsDir       = "wsl-root"
```

Comment:
Please update docs/internals.md too

Reply:
Also names seem confusing

Reply:
Now that I'm looking at this again, I can probably just use BaseDisk for WslRootFsDir (that directory just holds the a `.vhdx` virtual harddrive).

WslRootFs is the downloaded root file system that is imported by `wsl.exe --import` and transformed into the `.vhdx`.

Reply:
Fixed in latest revision